code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -1122,3 +1122,55 @@ mob/living/carbon/human/proc/change_monitor()
 		to_chat(src, span("notice", "You sense " + jointext(feedback, " ") + " towards the [dir2text(text2num(d))]."))
 	if(!length(dirs))
 		to_chat(src, span("notice", "You detect no psionic signatures but your own."))
+
+// flick tongue out to read gasses
+/mob/living/carbon/human/proc/tongue_flick()
+	set name = "Tongue-flick"
+	set desc = "Flick out your tongue to sense the gas in the room."
+	set category = "IC"
+
+	if(stat == DEAD)
+		return
+
+	if(last_special > world.time)
+		return
+
+	if(head && (head.body_parts_covered & FACE))
```

Comment:
Shouldn't this be mouth?

Reply:
From what I gleaned in code, there's no "MOUTH body_parts_covered" modifier. It could be that it just hasn't been used yet.